### ISSUE TITLE
Figure 7 caption, line 238, line 242: This whole section has many undefined concepts and issues. 

### ISSUE THREAD

**Issue by heroxbd** [heroxbd/waveform-analysis#113]
what is a "kernel = 21" ? what are number of channels? The behavior of DW depends on the assumed parameters of the pulse. The statement on line 238 is stated as a general statement. It is not a general result. What is "matching of waveforms horizontally" ? What is effective α̂ scaling ?

**Comment by heroxbd** [comment]
可以按照评审意见重新画个图吗？

**Comment by heroxbd** [comment]
与 https://github.com/heroxbd/waveform-analysis/issues/111 一起修改

**Comment by TakaraResearch** [comment]
我在另一个Issue地方写了一下 基础的东西 
把神经网络 概念 训练； 卷积神经网络 卷积 通道 都介绍了一下
我想大概是有用的

另外，我觉得在方法的地方应该包含类似于以下的内容：

In our study, One-Dimensional Convolutional Neural Networks (1D-CNN) are introduced to build up the photon prediction based on PMT waveforms. The network uses 1D kernels to build up local relationships in time, and output channels from each layer are later convolution calculated in the next layer to enable a mechanism of combining low-level patterns into high-level-patterns. 

在介绍具体网络的时候应该包含以下类似内容

A 5-layer 1D CNN is used.
The input of the network is a one channel PMT number changes in time dimension.
The output of the network is the photon prediction result as a one channel output with time.

另外 应使用
channel number/ number of channel  来取代channel
kernel length/size 取代kernel
来进行结构表达

另外 对于为什么这么选网络   对于一般性的一类问题，具体结构相对可变
可以说
The structure of the networks is relatively arbitrary and could be modified based on actual application and preferences.

**Comment by TakaraResearch** [comment]
另外，sparsity的问题
以下表达可能有益处
CDF 1D-wasserstein loss function is introduced to gurantee an always valid loss function with efficient gradient in all sparsity cases, so that the neural network training will succeed and converge to an accurate prediction.
